tf_big/python/ops/big_ops_test.py

```diff
@@ -13,10 +13,19 @@ def test_import_export(self):
     with tf.Session() as sess:
       inp = [[b"43424"]]
```

Comment:
note: we could probably get around the need for byte strings if we wrapped the ops in python land

Reply:
we could also do some shape matching, and eg automatically convert a scalar into a (1,1) matrix